app/views/application/_dashboardstats.html.erb

```diff
@@ -1,8 +1,14 @@
 <section class='dashboard-stats'>
 
+  <section class='yearly-revenue'>
+    <h3>Yearly Revenue</h3>
+    <%= bar_chart @yearly_revenue.group_by_month(:created_at, format: "%b %Y").count %>
+  </section>
+
   <section class='percent-sold'>
     <h3>Total Quantity of Items Sold</h3>
-    <p>Sold <%= Item.total_sold_quantity(@user) %> items, which is <%= Item.percent_sold(@user) %>% of your total inventory</p>
+    <p>Sold <%= @user.items.total_sold_quantity(@user) %> items, which is <%= @user.items.percent_sold(@user) %>% of your total inventory</p>
```

Comment:
again, calling `@instance.association.anything` is effectively doing a class method call, which we never want to do in a View